CONTRIBUTING.md

```diff
@@ -1,42 +1,51 @@
 
-[MonoGame Coding Guidelines]: https://github.com/mono/MonoGame/wiki/Coding-Guidelines
+The MonoGame Team has defined the simple guide below to help you as a new contributor to the project.  Please read this document carefully and try to conform to our process and guidelines as you help us improve and grow MonoGame.
 
-Etiquette
-=========
 
-In general, we do not accept pull requests that merely shuffle code around, split classes in multiple files, reindent the code or are the result of running a refactoring tool on the source code.  This is done for three reasons:
+## How To Contribute
 
-* We have our own coding guidelines
-* Some modules are imported from upstream sources and we want to respect their coding guidelines
-* It destroys valuable history that is often used to investigate bugs, regressions and problems
+MonoGame consists of a `master` branch with the latest stable releases and a `develop` branch with the day to day development changes. All new contributions and fixes should be directed to the `develop` branch.
 
-License
-=======
+The MonoGame project follows standard [GitHub flow](https://guides.github.com/introduction/flow/index.html).  You should learn and be familiar with how to [use Git](https://help.github.com/articles/set-up-git/), how to [create a fork of MonoGame](https://help.github.com/articles/fork-a-repo/), and how to [submit a Pull Request](https://help.github.com/articles/using-pull-requests/).
 
-The MonoGame project uses the Microsoft Public License.  See `LICENSE.txt` for more details.  Some third-party libraries used by MonoGame may be under a different license.  Please refer to those libraries for details on the license they use.
+After you submit a PR the [MonoGame build bot](https://github.com/mgbot?tab=activity) will automatically build your changes and verify all unit tests pass.  One or more of the project maintainers and other contributors will help review your changes and provide constructive feedback to improve your submission.
 
-Submitting Patches
-==================
+Once we are satisfied that your changes are good for MonoGame it will be merged.
 
-MonoGame consists of two primary branches:
 
-* `master` is the stable branch from which releases are made.  Pull requests direct to the master branch will not be accepted.
-* `develop` is the unstable development branch.  Pull requests must be made against the develop branch.
+## Quick Guidelines
 
-The process for making a pull request is generally as follows:
+Here are a few simple rules and suggestions to remember when contributing to MonoGame:
 
-1. Make a feature branch from `develop` for the change.
-2. Edit, build and test the feature.
-3. Commit to your local repository.
-4. Push the feature branch to your GitHub fork.
-5. Create the pull request.
+* :bangbang: **NEVER** commit code that you didn't personally write.
+* :bangbang: **NEVER** use decompiler tools to steal code and submit them as your own work.
```

Comment:
Should there also be something more explicit on studying decompiled XNA code? If I've understood correctly, that's prohibited as well, since we are doing strictly black box design.